docs/database-access/accessing-the-database-access-menu-for-configuring-and-monitoring/quickstart.yml

```diff
@@ -14,18 +15,18 @@ spec:
   description: 'A quick step-by-step guide for system administrators on how to access the **Database Access** menu, so they can import a provider account, provision a new database instance, and to monitor database instances and applications.'
-  prerequisites: ["Access to the OpenShift console.", "A service account with either the [MongoDB Atlas](https://www.mongodb.com/atlas/database), or [Crunchy Data Bridge](https://www.crunchydata.com), or [CockroachDB](https://www.cockroachlabs.com) cloud database provider."]
+  prerequisites: ["Access to the OpenShift console.", "Installation of the Red Hat OpenShift Database Access add-on.", "A service account with either the [MongoDB Atlas](https://www.mongodb.com/atlas/database), or [Crunchy Data Bridge](https://www.crunchydata.com), or [CockroachDB](https://www.cockroachlabs.com), or [Amazon RDS](https://portal.aws.amazon.com/billing/signup?nc2=h_ct&src=header_signup&redirect_url=https%3A%2F%2Faws.amazon.com%2Fregistration-confirmation#/start/email) cloud-hosted database provider."]
```

Comment:
According to IBM Style: 'Use ["either"] Use to mean "any one of two".'

You might want to remove the word "either", and use a nested bulleted list:
"A service account with one of the following cloud database providers:"

Reply:
Fair point.  I need to revisit this.  Awhile back, I tried to use a bulleted list, and it broke something with the formatting of the prereqs in the quick starts, so I need to take a look and try again.

Reply:
Ah yes, it might be easier to create the nested list when you have the separate README.adoc file. 

In the meantime, you could make the other changes (remove "either", and change the lead-in sentence), and just use the comma-separated list instead of a nested bulleted list? Not urgent though -- just something to think about next time you're updating :)